### DISCUSSION TITLE
能支持docker部署吗？

### DISCUSSION THREAD

**Discussion by kokojacket** [yuudi/rclone-webui-angular#21] [General]
小白一枚，不会代码。求支持docker部署

**Comment by yuudi** [comment]
部署方式和 rclone 一样，官方有 docker 镜像 rclone/rclone 可以直接使用，然后按照 [说明](https://github.com/yuudi/rclone-webui-angular/blob/master/docs/zh/Instructions.md) 配置即可

注意，即使使用 docker 部署，你仍然需要配置反向代理和 https 证书

**Comment by yuudi** [comment]
this issue will be moved to discussion

**Comment by yuudi** [comment]
提供一个命令行供参考

```sh
docker run -d --rm -v $HOME/.config/rclone:/config/rclone -p 127.0.0.1:5572:5572 rclone/rclone rcd --rc-serve --rc-user=用户名 --rc-pass=密码 --rc-addr=0.0.0.0:5572
```
